src/dosbox.cpp

```diff
@@ -597,19 +597,23 @@ void DOSBOX_Init(void) {
 	Pbool = secprop->Add_bool("nosound",Property::Changeable::OnlyAtStart,false);
 	Pbool->Set_help("Enable silent mode, sound is still emulated though.");
 
-	Pint = secprop->Add_int("rate",Property::Changeable::OnlyAtStart,44100);
+	Pint = secprop->Add_int("rate", Property::Changeable::OnlyAtStart, 44100);
```

Comment:
Shorter-hand references for `Property::Changeable::OnlyAtStart` might exist further up (maybe called `onstart` or `atstart`). Suggest dropping those in.